tests/forms_tests/field_tests/test_booleanfield.py

```diff
@@ -57,3 +57,7 @@ def test_booleanfield_changed(self):
         self.assertFalse(f.has_changed(True, 'True'))
         self.assertTrue(f.has_changed(False, 'True'))
         self.assertTrue(f.has_changed(True, 'False'))
+
+    def test_disabled_has_changed(self):
+        f = BooleanField(disabled=True)
+        self.assertFalse(f.has_changed('x', 'y'))
```

Comment:
It's better to use `assertIs(..., False)` since `assertFalse` will also pass if `bool(result) is False`.